lib/database.js

```diff
@@ -278,7 +278,6 @@ class Database {
   async scalar(table, field, ...conditions) {
     const row = await this.row(table, [field], ...conditions);
     const values = Object.values(row);
-    if (values.length < 1) return undefined;
```

Comment:
Not related to the consistent-return, but frankly speaking we can't make decision on this before deep investigation, decompiling and profiling. I am of with both solutions.

Reply:
This method does not comply with the recently renewed metarhia eslint rules and an error occurs as 'async scalar method expected no return value' by linter.
In addition, the element returned from the 0th index of an array with no elements will still be undefined, so we get rid of a really useless if.